src/FSharpx.Collections/Collections.fs

```diff
@@ -230,6 +230,13 @@ module Seq =
               notFirst := true
       } 
 
+    /// The catOptions function takes a list of Options and returns a seq of all the Some values.
+    let catOptions (xs:seq<Option<'a>>) =
+        seq {
```

Comment:
yes, but that's already two variables that have to have parentheses around them -- `catOptions` is supposed to be only a small helper function, I wrote it because I've found myself writing `(Seq.choose id)` way to often. [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)